package.json

```diff
@@ -12,26 +12,33 @@
     "build:2-compile": "tsc",
     "clean": "rm -rf build",
     "db:export": "tsx src/scripts/export.ts",
+    "dev": "prisma generate > /dev/null && tsx --no-warnings=ExperimentalWarning --watch src/main.ts",
     "db:format": "prisma format",
     "db:generate": "prisma generate",
     "db:push": "prisma db push",
     "db:seed": "tsx src/scripts/seed.ts",
     "db:studio": "prisma studio",
-    "dev": "prisma generate > /dev/null && tsx --no-warnings=ExperimentalWarning --watch src/main.ts",
     "format": "biome check .",
-    "format:fix": "biome format --write .",
+    "format:fix": "biome check --apply .",
     "lint": "biome lint .",
     "lint:fix": "biome lint . --apply",
     "nanoid": "nanoid",
     "seed": "tsx src/db/seed.ts",
     "start": "tsx src/main.ts",
     "test": "vitest",
-    "typecheck": "tsc --noEmit"
+    "typecheck": "tsc --noEmit",
+    "queue": "tsx --no-warnings=ExperimentalWarning --watch src/queue.ts"
```

Comment:
If it's a dev script I guess it should be called differently

Reply:
no idea tbh, I'm terrible at naming things and invalidating cache
https://github.com/l2beat/token-db/pull/31/commits/f64886124f70b8adfae58af550a881f2ca9d5c92